casts/free-podcasts-screencasts-en.md

```diff
@@ -110,7 +110,7 @@
 
 * [All Things Git](https://www.allthingsgit.com) - Edward Thomson and Martin Woodward (podcast)
 * [GitMinutes](http://www.gitminutes.com) (podcast)
-* [Jhan Carlos Silva - Git and Github Tutorial](https://www.youtube.com/watch?v=NPRUsCcZwJ4) (screencast)
+* [Jhan Carlos Silva - Git and GitHub Tutorial](https://www.youtube.com/watch?v=NPRUsCcZwJ4) (screencast)
```

Comment:
source uses 'Github'

Reply:
Reverted in 9d12e9596a95dc45ea0370a8258e5fad03c8477a